src/librustc/session/config.rs

```diff
@@ -1412,6 +1412,18 @@ pub fn default_configuration(sess: &Session) -> ast::CrateConfig {
     if sess.opts.crate_types.contains(&CrateTypeProcMacro) {
         ret.insert((Symbol::intern("proc_macro"), None));
     }
+    /*if nightly_options::is_nightly_build() {
```

Comment:
Stray commented block?

Reply:
I know this PR is currently broken. Will fix it soon.